### PR TITLE
fix most skipped tests & increase coverage

### PR DESCRIPTION
**Pull request by indiVar0508**

### CODE REVIEW

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
           pip install mkdocs==1.4.2
           pip install mkdocs-autorefs==0.4.1
           pip install mkdocs-awesome-pages-plugin==2.8.0
-          pip install mkdocstrings==0.19.0
+          pip install mkdocstrings['python']==0.19.0
```
Comment (aditya051293) [inline .github/workflows/pages.yml:22]: typo in commit message
"docuemntation" -> "documentation"

File: tests/inheritance/test_concrete_inheritance.py [hunk 2]
```diff
@@ -56,11 +56,11 @@ def test_each_class_has_distinct_version_class(self):
         assert self.ArticleVersion.__table__.name == "article_version"
         assert self.BlogPostVersion.__table__.name == "blog_post_version"
 
-    @mark.skipif("True")
-    def test_each_object_has_distinct_version_class(self):
-        article = self.Article()
-        blogpost = self.BlogPost()
-        textitem = self.TextItem()
+    @mark.skipif("True", reason="concrete property is not supported yet")
```
Comment (aditya051293) [inline tests/inheritance/test_concrete_inheritance.py:62]: Why are we removing the reason?

Comment (indiVar0508) [reply tests/inheritance/test_concrete_inheritance.py:62]: kept the tests but excluded the this test as for now we do not support concrete props

File: sqlalchemy_history/utils.py [hunk 3]
```diff
@@ -133,10 +133,7 @@ def version_class(model):
     :returns:
     """
     manager = get_versioning_manager(model)
-    try:
-        return manager.version_class_map[model]
-    except KeyError:
-        return model
+    return manager.version_class_map.get(model, None)
```
Comment (aditya051293) [inline sqlalchemy_history/utils.py:136]: In relationship_builder we have
```
        try:
            self.remote_cls = version_class(self.property.mapper.class_)
            self.versioned = True
        except (AttributeError, KeyError):
            return
```
Now we won't get KeyError, We would want to make the changes over there as well.

File: tests/utils/test_tx_column_name.py [hunk 4]
```diff
@@ -21,3 +21,19 @@ def test_with_versioned_class(self):
...
 
 create_test_cases(TxColumnNameTestCase, setting_variants=setting_variants)
+
+
+setting_variants = {
+    "versioning_strategy": ["validity"],
+}
+
+
+class TxColumnNameTestCaseWithValidity(TestCase):
```
Comment (aditya051293) [inline tests/utils/test_tx_column_name.py:31]: Why this test case getting removed in this commit?

File: tests/relationships/test_many_to_many_relations.py [hunk 5]
```diff
@@ -306,9 +303,6 @@ def test_single_insert(self):
         assert article.versions[0] in reference1.versions[0].cited_by
 
     def test_multiple_inserts_over_multiple_transactions(self):
-        if self.driver == "mysql" and self.connection.dialect.server_version_info < (5, 6):
```
Comment (aditya051293) [inline tests/relationships/test_many_to_many_relations.py:309]: Can we add it in commit message the reason for removing it.

File: tests/test_exotic_operation_combos.py [hunk 6]
```diff
@@ -16,8 +18,9 @@ def test_insert_deleted_object(self):
         self.session.add(article2)
         self.session.commit()
         assert article2.versions.count() == 2
-        assert article2.versions[0].operation_type == 0
-        assert article2.versions[1].operation_type == 1
+        assert article2.versions[0].operation_type == Operation.INSERT
+        assert article2.versions[1].operation_type == Operation.UPDATE
+        assert article2.versions[1].operation_type != Operation.INSERT
```
Comment (aditya051293) [inline tests/test_exotic_operation_combos.py:23]: Can we remove assert, looks redundant

**Review by aditya051293 — APPROVED** [review]

**Pull request merged**